config/jail.conf

```diff
@@ -277,6 +277,14 @@ logpath  = %(apache_error_log)s
 maxretry = 2
 
 
+[apache-fakegooglebot]
+
+port     = http,https
+logpath  = %(apache_access_log)s
+maxretry = 1
+ignorecommand = %(ignorecommands_dir)s/apache-fakegooglebot.sh <ip>
```

Comment:
as I have mentioned in another issue, better to not ship implementation specific suffixes here (i.e.  remove ```.sh``` from the name of that command i.e. rename the file as well)